scripts/cluster-upgrade.sh

```
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Variable sanity check
+if [[ -z ${PGVERSIONOLD} ]]; then
+  echo "Environment variable PGVERSIONOLD is empty."
+  echo "It must be set to postgresql version of the old cluster."
```

On second thought, I think it would be good to fail early in this case.